packages/ui/src/providers/ComponentMap/buildComponentMap/mapFields.tsx

```diff
@@ -200,6 +200,7 @@ export const mapFields = (args: {
               : undefined,
           labels: 'labels' in field ? field.labels : undefined,
           options: 'options' in field ? field.options : undefined,
+          relationTo: 'relationTo' in field ? field.relationTo : undefined,
```

Comment:
This corrects problems in other tests -- it may be worth extract this change out to its own PR so no one has to wait for this full suite to pass to get the update.